src/models/recovered_msg.rs

```diff
@@ -61,12 +61,12 @@ impl RecoveredMessage {
     let result = task::spawn_blocking(move || {
       use crate::schema::recovered_msgs::dsl::*;
 
-      let conn = conn.lock().unwrap();
+      let mut conn = conn.lock().unwrap();
```

Comment:
An error will only be returned from `Mutex::lock` if another thread panicked while holding the mutex. So this thread would also panic (by calling `unwrap`), but that will not affect uptime or cause a panic in the main thread. When we await the future returned by `spawn_blocking`, it will return a JoinError if a panic occurred in the blocking thread, which we can handle gracefully (by converting it to a PgStoreError).